packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -362,12 +371,49 @@ const GlobalHeader = ({ className, activeSite }) => {
                   }
                 `}
               >
-                <Icon css={actionIcon} name="fe-search" size="0.875rem" />
+                <Icon css={actionIcon} name="fe-search" size="1.5rem" />
               </Link>
               <SearchInput
                 placeholder={t('searchInput.placeholder')}
-                size={SearchInput.SIZE.SMALL}
+                size={SearchInput.SIZE.MEDIUM}
                 focusWithHotKey="/"
+                style={{
+                  input: css`
+                    --icon-color: red;
+                    --hover-border-color: var(--color-neutrals-600);
+                    --background-color: var(--color-neutrals-700);
+                    background: var(--background-color);
+                    border: 1px solid trasparent;
+                    border-style: none;
+                    height: 40px;
+
+                    .dark-mode & {
+                      --background-color: var(--color-dark-500);
+                    }
+
+                    &:hover {
+                      --hover-border-color: var(--color-neutrals-600);
+                      border-style: solid;
+                      border-color: --hover-border-color;
```

Comment:
I noticed that the placeholder text "wiggles" a little bit when hovering over the input. This is (probably) because the border is pushing text over. If we want to avoid this, we could use an [inset `box-shadow`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow) to achieve the same effect:
```suggestion
                      box-shadow: inset 0 0 0 2px var(--hover-border-color);
}
```